### DISCUSSION TITLE
Python 3.11 released with builtin toml support.

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1580] [General]
Python 3.11 has been released with speed improvements and builtin toml support: https://docs.python.org/3/whatsnew/3.11.html

- Sounds like VisiData should add support for toml, too.
- The github actions  need updating to use 3.11: https://github.com/saulpw/visidata/blob/develop/.github/workflows/main.yml
   - Also, should the top level README.md file point at the GitHub action rather than the circleci link?

@anjakefala @saulpw

**Comment by anjakefala** [comment]
> The github actions need updating to use 3.11: https://github.com/saulpw/visidata/blob/develop/.github/workflows/main.yml

>    Also, should the top level README.md file point at the GitHub action rather than the circleci link?

Yup! Open an issue, please?

**Comment by frosencrantz** [comment]
Three issues created: new action, update readme, add toml support.  I think that was all the items... :)

**Comment by anjakefala** [comment]
Thanks to @ajkerrigan, we now have toml support. :tada: 

https://github.com/saulpw/visidata/pull/1894

**Reply by frosencrantz** [reply]
Thanks, @ajkerrigan!
